samples/bluetooth/broadcaster/src/main.c

```diff
@@ -0,0 +1,62 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <sys/printk.h>
+#include <sys/util.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/hci.h>
+
+static uint8_t mfg_data[] = { 0xff, 0xff, 0x00 };
+
+static const struct bt_data ad[] = {
+	BT_DATA(BT_DATA_MANUFACTURER_DATA, mfg_data, 3),
+};
+	
+void main(void)
+{
+	int err;
+
+	printk("Starting Advertiser\n");
+
+	/* Initialize the Bluetooth Subsystem */
+	err = bt_enable(NULL);
+	if (err) {
+		printk("Bluetooth init failed (err %d)\n", err);
+		return;
+	}
+
+	printk("Bluetooth initialized\n");
+
+	do {
+		k_msleep(1000);
+
+                printk("Sending advertising data: 0x%02X \n", mfg_data[2]);
```

Comment:
Indentation, use tabs,

```suggestion
		k_msleep(1000);

		printk("Sending advertising data: 0x%02X \n", mfg_data[2]);
```